src/components/landing/PSEBanner.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Button from "@/components/landing/Button";
+import PSEBg from "@/public/landing/pse_bg.webp";
+import Image from "next/image";
+
+const PSEBanner = () => {
+  return (
+    <div
+      className={`relative flex h-[309px] w-full flex-col items-center justify-center overflow-clip transition-transform duration-300 ease-out sm:h-[359px] md:h-[409px] lg:h-[459px]`}
```

Comment:
breakpoints work for the size they're applied at and up. for example, if you want to set a default size for smaller screens and then have medium and larger screens have a larger height for the element, it would look like this:
```suggestion
      className={`relative flex h-1/5 w-full flex-col items-center justify-center overflow-clip transition-transform duration-300 ease-out md:h-1/2`}
```